packages/toolkit/src/view/pipeline/view-pipeline/PipelineRunViews/Run.tsx

```diff
@@ -72,9 +73,32 @@ export const PipelineRunView = ({ id, pipeline }: PipelineRunProps) => {
   const [outputActiveView, setOutputActiveView] =
     React.useState<PipelineOutputActiveView>("preview");
 
+  const [codeBlockMaxHeight, setCodeBlockMaxHeight] = React.useState("auto");
```

Comment:
@orangecoloured Could you help me add why we need this feature as comment~

Reply:
Added comment